Rakefile

```diff
@@ -38,14 +38,31 @@ RSpec::Core::RakeTask.new(:acceptance) do |t|
   t.pattern = 'spec/acceptance'
 end
 
-task :metadata do
+task :metadata_lint do
   sh "metadata-json-lint metadata.json"
 end
 
+desc "Custom: Download third-party modules"
+task :r10k do
+  system 'r10k puppetfile install -v'
+end
+
+desc "Custom: Prepare symbolic link to root puppet module folders"
+task :symlink do
+  FileUtils.remove_dir("development/modules/chocolatey_server",:verbose => true)
+  FileUtils.mkdir_p "development/modules/chocolatey_server"
+  FileUtils.cd "development/modules/chocolatey_server"
+  system 'ln -s "../../../manifests" "manifests"'
+end
+
```

Comment:
Not sure about this, but no harm in pulling it in.

Reply:
I had lot of dependencies to windows puppet modules and usually some of them have custom facter variables so with `vagrant` and the puppet provisioner would not work.   I will try to give it a shot with the puppet provisioner an get back to you.